hbook/ntup/ntup_test/ntup_example_test.go

```diff
@@ -0,0 +1,58 @@
+package ntup_test
+
+import (
```

Comment:
you are missing imports for the following packages:

- `golang.org/x/epx/rand`
- `gonum.org/v1/gonum/stat/distuv`
- `go-hep.org/x/hep/hbook`

and you have packages that are imported but not used:
- `log`, `reflect`, `testing`.